source/includes/api/_dataset.md

```diff
@@ -1,15 +1,17 @@
 # Dataset
 
-## What is a Dataset?
+## What is a dataset?
 
-If you are new to the RW API, or want to learn more about the concept of a dataset, we strongly encourage you to read the [dataset concept](#dataset) documentation first. It gives you a brief and clear description of what a dataset is, and what it can do for you. 
+If you are new to the RW API, or want to learn more about the concept of a dataset, we strongly encourage you to read the [dataset concept](#dataset) documentation first. It gives you a brief and clear description of what a dataset is, and what it can do for you.
 
-Once you've read that section, you can come back here to learn more details about [using existing datasets](#getting-all-datasets) or [creating your own](#creating-a-dataset).
+Once you've read that section, you can come back here to learn more details about using the RW API's datasets feature. The RW API is home to many datasets uploaded by [WRI](https://www.wri.org/), its partner organizations, or by API users like you. These datasets contain a lot of information about a wide range of topics that you may want to learn about or build on top of. To find out more about finding and accessing the datasets already available on the RW API, check out the documentation on [getting datasets](#getting-all-datasets).
 
-## Getting all datasets
+You can also [create you own datasets](#creating-a-dataset) on the RW API, if you'd like to share your data with the world, or 
+ if you are looking to use the RW API and its features to gain insights into your data.
 
-This endpoint will allow you to get the list of the datasets available in the API. You can customize the result in multiple ways, using the options described below.
+## Getting all datasets
 
+This endpoint will allow you to get the list of the datasets available in the API, and it's a great place for new users to start exploring the RW API. By default, this endpoint will give you a paginated list of 10 datasets. In the sections below, we'll explore how you can customize this endpoint call to match your needs. 
 
 ```shell
 curl -X GET https://api.resourcewatch.org/v1/dataset
```

Comment:
We need documentation here for the schema of the dataset object. Fortunately, should be similar to the search attributes table below.

Reply:
so this is a simple question with two very different answers: do we want to have per-endpoint schemas, or a more global, complete one?
- if we go with a per-endpoint approach, we'll basically be duplicating a lot of info, and making it more troublesome to update. It will also mean we're more prone to errors going forward
- the approach I'd like to try is to centralize this in a single place (the newly added "dataset reference" section) and then point users to it from the different endpoints. It's a bit less accurate as, for example, the reference may have fields that are not exposed on gets, or that may only for internal usage only. 

Let me know if you prefer the first approach instead

Reply:
Second approach makes more sense to me. But let's link it from here